{
  "_content": {
    "Summary": {
      "children": [
        {
          "type": "Paragraph",
          "data": {
            "inline": [
              {
                "type": "Words",
                "data": {
                  "value": "A plugin for the client allowing replay of remote exceptions locally"
                }
              }
            ],
            "inner": []
          }
        }
      ],
      "title": null
    },
    "Extended Summary": {
      "children": [
        {
          "type": "Paragraph",
          "data": {
            "inline": [
              {
                "type": "Words",
                "data": {
                  "value": "Adds the following methods (and their async variants)to the given client:"
                }
              }
            ],
            "inner": []
          }
        },
        {
          "type": "Paragraph",
          "data": {
            "inline": [
              {
                "type": "Words",
                "data": {
                  "value": "- "
                }
              },
              {
                "type": "Verbatim",
                "data": {
                  "value": [
                    "recreate",
                    "_",
                    "error",
                    "_",
                    "locally"
                  ]
                }
              },
              {
                "type": "Words",
                "data": {
                  "value": ": main user method - "
                }
              },
              {
                "type": "Verbatim",
                "data": {
                  "value": [
                    "get",
                    "_",
                    "futures",
                    "_",
                    "error"
                  ]
                }
              },
              {
                "type": "Words",
                "data": {
                  "value": ": gets the task, its details and dependencies,"
                }
              }
            ],
            "inner": []
          }
        },
        {
          "type": "Paragraph",
          "data": {
            "inline": [
              {
                "type": "Words",
                "data": {
                  "value": "    responsible for failure of the given future."
                }
              }
            ],
            "inner": []
          }
        }
      ],
      "title": null
    },
    "Parameters": {
      "children": [],
      "title": null
    },
    "Returns": {
      "children": [],
      "title": null
    },
    "Yields": {
      "children": [],
      "title": null
    },
    "Receives": {
      "children": [],
      "title": null
    },
    "Raises": {
      "children": [],
      "title": null
    },
    "Warns": {
      "children": [],
      "title": null
    },
    "Other Parameters": {
      "children": [],
      "title": null
    },
    "Attributes": {
      "children": [],
      "title": null
    },
    "Methods": {
      "children": [],
      "title": null
    },
    "Notes": {
      "children": [],
      "title": null
    },
    "Warnings": {
      "children": [],
      "title": null
    }
  },
  "refs": [],
  "ordered_sections": [
    "Summary",
    "Extended Summary"
  ],
  "item_file": "/Users/bussonniermatthias/dev/distributed/distributed/recreate_exceptions.py",
  "item_line": 54,
  "item_type": "<class 'type'>",
  "aliases": [
    "distributed.recreate_exceptions.ReplayExceptionClient"
  ],
  "example_section_data": {
    "children": [],
    "title": null
  },
  "see_also": [],
  "version": "2021.01.1+9.g613b3c820",
  "signature": "ReplayExceptionClient(client)",
  "references": null,
  "logo": "logo.png",
  "qa": "distributed.recreate_exceptions.ReplayExceptionClient",
  "arbitrary": []
}